var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -144,9 +145,27 @@ def check(self):
         "5.2.0",
     ]:
         depends_on("hip@" + ver, when="@" + ver)
-        depends_on("llvm-amdgpu@" + ver, type="build", when="@" + ver)
         depends_on("rocminfo@" + ver, type="build", when="@" + ver)
 
+    for ver in [
+        "3.9.0",
+        "3.10.0",
+        "4.0.0",
+        "4.1.0",
+        "4.2.0",
+        "4.3.0",
+        "4.3.1",
+        "4.5.0",
+        "4.5.2",
+        "5.0.0",
+        "5.0.2",
+        "5.1.0",
+        "5.1.3",
+        "5.2.0",
+    ]:
+        depends_on("llvm-amdgpu@" + ver, type="build", when="@" + ver)
+        depends_on("rocm-openmp-extras@" + ver, type="build", when="@" + ver)
+
```

Comment:
Why is this change needed? rocBLAS depends on LLVM due to Tensile, but I don't think OpenMP is required for the base rocBLAS library. OpenMP is only used in the tests.

Reply:
that was a copy-pasta error, I was just pulling it up since the versions needed to match. The new loop is because rocm-openmp-extras doesn't have the full version range.